### ISSUE TITLE
Hue Lights Error

### ISSUE THREAD

**Issue by NickWaterton** [spacemanspiff2007/HABApp#74]
Got the following error when turning on some Hue lights:

```

[2019-09-28 01:51:49,395] [ HABApp.openhab.Connection]    ERROR |        on_sse_event: '>' not supported between instances of 'str' and 'float'
[2019-09-28 01:51:49,396] [ HABApp.openhab.Connection]    ERROR |        on_sse_event: Traceback (most recent call last):
[2019-09-28 01:51:49,396] [ HABApp.openhab.Connection]    ERROR |        on_sse_event:   File "/home/nick/Scripts/HABApp/HABApp/openhab/oh_connection.py", line 120, in on_sse_event
[2019-09-28 01:51:49,396] [ HABApp.openhab.Connection]    ERROR |        on_sse_event:     HABApp.core.Items.get_item(event.name).set_state(event.value)
[2019-09-28 01:51:49,397] [ HABApp.openhab.Connection]    ERROR |        on_sse_event:   File "/home/nick/Scripts/HABApp/HABApp/core/items/item_color.py", line 27, in set_state
[2019-09-28 01:51:49,397] [ HABApp.openhab.Connection]    ERROR |        on_sse_event:     self.hue = min(max(0.0, hue), HUE_FACTOR)
[2019-09-28 01:51:49,397] [ HABApp.openhab.Connection]    ERROR |        on_sse_event: TypeError: '>' not supported between instances of 'str' and 'float'
```

**Comment by spacemanspiff2007** [comment]
Do you have a unit configured? Could you please provide the item config?

**Comment by NickWaterton** [comment]
These are my hue lights (only two), plus two hue tap switches. All of these were activated at 1:51 last night, so I don't know which caused the error.

The hue lights are white lights, not colour.
OH items:
```
/* Bedside Lux Lights */
Dimmer JillBedsideDim     "Jill's Bedside"  (hueWhites, FF_Master_Bedroom, Lights)   {channel="hue:0100:1:bulb2:brightness"}
Dimmer NickBedsideDim     "Nick's Bedside"  (hueWhites, FF_Master_Bedroom, Lights)   {channel="hue:0100:1:bulb1:brightness"}
/* Colour Lights */
Color hueBloom     "Bloom"            <colorlight>   (hueColours, GF_Family, Lights)               ["Lighting"]  {channel="hue:0200:1:bulb3:color"}
Color hueIris      "Fireplace Light"  <colorlight>   (hueColours, GF_Family, Lights, bedtime)      ["Lighting"]  {channel="hue:0200:1:bulb4:color"}   //fireplace light
```

**Comment by NickWaterton** [comment]
I have played about, trying to reproduce what happened last night with the hue lights, but no luck.

I'll wait and see if the same thing happens tonight, or if it was tied into the hue lights being turned off (by the dog).

Is it possible this line is the culprit?
```
`bedtime?.members.forEach[ item,i|createTimer(now.plusMillis(i*100)) [ sendCommand(item, OFF) ] ]
```
The iris colour light is part of the `bedtime` group, so it get sent the command 'OFF' as part of the bedtime routine. It was OFF at the time the command was sent (but powered up).

**Comment by spacemanspiff2007** [comment]
If you could post the state of the item (at the time) from the openhab event log I can try to reproduce the issue.

**Comment by NickWaterton** [comment]
OK, I found it:
The command that causes this is:
`sendCommand(hueIris, OFF)`

The item is:
`hue.items:Color hueIris      "Fireplace Light"  <colorlight>   (hueColours, GF_Family, Lights, bedtime)      ["Lighting"]  {channel="hue:0200:1:bulb4:color"}   //fireplace light`

Apparently HABApp doesn't like sending `OFF` to a Color item (even though this works).

Event.log shows:
```2019-09-30 09:59:07.325 [ome.event.ItemCommandEvent] - Item 'hueIris' received command OFF
2019-09-30 09:59:07.327 [nt.ItemStatePredictedEvent] - hueIris predicted to become OFF
```

**Comment by spacemanspiff2007** [comment]
Fixed with 0.9.0
